content/millennium/r4/entities/location.md

```diff
@@ -53,9 +53,16 @@ Search for Locations that meet supplied query parameters:
 
 ### Parameters
 
- Name  | Required? | Type      | Description
--------|-----------|-----------|-------------------------------------------------------
- `_id` | Yes       | [`token`] | The logical resource id associated with the resource.
+ Name            | Required?               | Type       | Description
+-----------------|-------------------------|------------|-------------------------------------------------------
+ `_id`           | This or `-physicalType` | [`token`]  | The logical resource id associated with the resource.
+ `-physicalType` | This or `_id`           | [`token`]  | The location’s physical type. Example: `http://terminology.hl7.org/CodeSystem/location-physical-type|ro`
```

Comment:
just wanted to make sure `|ro` is not a typo?

Reply:
Its from this codeSet https://www.hl7.org/fhir/valueset-location-physical-type.html meaning room

Reply:
cool